src/deploy/table_location.sql

```diff
@@ -0,0 +1,52 @@
+BEGIN;
+
+CREATE TABLE maevsi.location (
+  id            UUID PRIMARY KEY,
+
+  location_type CHAR(1) NOT NULL,
+  latitude      DOUBLE PRECISION NOT NULL,
+  longitude     DOUBLE PRECISION NOT NULL,
+
+  -- -- possible future extension using PostGIS
+  -- geom          GEOMETRY
+
+  CHECK (location_type IN ('A', 'E'))
+);
+
+COMMENT ON TABLE maevsi.location IS 'Location data based on GPS coordnates.';
+COMMENT ON COLUMN maevsi.location.id IS 'The locations''s internal id.';
+COMMENT ON COLUMN maevsi.location.location_type IS 'The type of the location (A = account, E = event)';
+COMMENT ON COLUMN maevsi.location.latitude IS 'The coordinate''s latitude.';
+COMMENT ON COLUMN maevsi.location.longitude IS 'The coordinate''s longitude.';
+
+GRANT SELECT, INSERT, UPDATE, DELETE ON TABLE maevsi.location TO maevsi_account;
+GRANT SELECT ON TABLE maevsi.location TO maevsi_anonymous;
```

Comment:
I think the RLS policies are missing. Right now any logged in user could delete all locations, no?